Code/Engine/RendererDX11/Device/Implementation/DeviceDX11.cpp

```diff
@@ -3,6 +3,8 @@
 #include <Core/System/Window.h>
 #include <Foundation/Basics/Platform/Win/IncludeWindows.h>
 #include <Foundation/Configuration/Startup.h>
+#include <RendererCore/Pipeline/Declarations.h>

```

Comment:
Renderer implementations should only depend on RendererFoundation not on RendererCore

Reply:
Were not in use, removed.